Cleanup java build issues

```bash
# Cleanup git
git clean -idfx -e .idea/ -e .vscode/

# Cleanup gradle build cache
rm -r $HOME/.gradle/caches/


./gradlew --version
------------------------------------------------------------
Gradle 8.5
------------------------------------------------------------

Build time:   2023-11-29 14:08:57 UTC
Revision:     28aca86a7180baa17117e0e5ba01d8ea9feca598

Kotlin:       1.9.20
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          21.0.8 (Ubuntu 21.0.8+9-Ubuntu-0ubuntu124.04.1)
OS:           Linux 6.8.0-84-generic amd64


❯ ./gradlew cli

> Task :compileJava
/home/nyurik/.gradle/caches/modules-2/files-2.1/io.github.sebasbaumh/mapbox-vector-tile-java/23.1.0/12a75773cd87a7e533fcd833f4f030267b1dd7ee/mapbox-vector-tile-java-23.1.0.jar(/io/github/sebasbaumh/mapbox/vectortile/adapt/jts/MvtReader.class): warning: Cannot find annotation method 'value()' in type 'NonNullByDefault': class file for org.eclipse.jdt.annotation.NonNullByDefault not found
warning: unknown enum constant DefaultLocation.PARAMETER
  reason: class file for org.eclipse.jdt.annotation.DefaultLocation not found
warning: unknown enum constant DefaultLocation.RETURN_TYPE
/home/nyurik/.gradle/caches/modules-2/files-2.1/io.github.sebasbaumh/mapbox-vector-tile-java/23.1.0/12a75773cd87a7e533fcd833f4f030267b1dd7ee/mapbox-vector-tile-java-23.1.0.jar(/io/github/sebasbaumh/mapbox/vectortile/adapt/jts/TagKeyValueMapConverter.class): warning: Cannot find annotation method 'value()' in type 'NonNullByDefault'
warning: unknown enum constant DefaultLocation.PARAMETER
warning: unknown enum constant DefaultLocation.RETURN_TYPE
/home/nyurik/.gradle/caches/modules-2/files-2.1/io.github.sebasbaumh/mapbox-vector-tile-java/23.1.0/12a75773cd87a7e533fcd833f4f030267b1dd7ee/mapbox-vector-tile-java-23.1.0.jar(/io/github/sebasbaumh/mapbox/vectortile/adapt/jts/model/JtsMvt.class): warning: Cannot find annotation method 'value()' in type 'NonNullByDefault'
warning: unknown enum constant DefaultLocation.PARAMETER
warning: unknown enum constant DefaultLocation.RETURN_TYPE
/home/nyurik/.gradle/caches/modules-2/files-2.1/io.github.sebasbaumh/mapbox-vector-tile-java/23.1.0/12a75773cd87a7e533fcd833f4f030267b1dd7ee/mapbox-vector-tile-java-23.1.0.jar(/io/github/sebasbaumh/mapbox/vectortile/adapt/jts/ITagConverter.class): warning: Cannot find annotation method 'value()' in type 'NonNullByDefault'
warning: unknown enum constant DefaultLocation.PARAMETER
warning: unknown enum constant DefaultLocation.RETURN_TYPE
/home/nyurik/.gradle/caches/modules-2/files-2.1/io.github.sebasbaumh/mapbox-vector-tile-java/23.1.0/12a75773cd87a7e533fcd833f4f030267b1dd7ee/mapbox-vector-tile-java-23.1.0.jar(/io/github/sebasbaumh/mapbox/vectortile/adapt/jts/model/JtsLayer.class): warning: Cannot find annotation method 'value()' in type 'NonNullByDefault'
warning: unknown enum constant DefaultLocation.PARAMETER
warning: unknown enum constant DefaultLocation.RETURN_TYPE
warning: [rawtypes] found raw type: Vector
  missing type arguments for generic class Vector<T,K>
  where T,K are type-variables:
    T extends Buffer declared in class Vector
    K extends Object declared in class Vector
/home/nyurik/dev/maplibre/maplibre-tile-spec/java/src/main/java/org/maplibre/mlt/decoder/StringDecoder.java:215: warning: [deprecation] decode(byte[],int[],byte[]) in FsstEncoder has been deprecated
          FsstEncoder.decode(
                     ^
17 warnings
```

--- a/java/mlt-core/src/main/java/org/maplibre/mlt/decoder/StringDecoder.java
+++ b/java/mlt-core/src/main/java/org/maplibre/mlt/decoder/StringDecoder.java
@@ -323,7 +323,10 @@ public class StringDecoder {
     var decompressedLength = length.stream().mapToInt(i -> i).sum();
     var values =
         FsstEncoder.decode(
-            symbols, symbolLength.stream().mapToInt(i -> i).toArray(), compressedCorpus, decompressedLength);
+            symbols,
+            symbolLength.stream().mapToInt(i -> i).toArray(),
+            compressedCorpus,
+            decompressedLength);
     var decodedData = IntegerDecoder.decodeIntStream(data, dataOffset, dataMetadata, false);
 
     var decodedDictionary = new ArrayList<String>();

--- a/java/mlt-core/src/main/java/org/maplibre/mlt/vector/FeatureTable.java
+++ b/java/mlt-core/src/main/java/org/maplibre/mlt/vector/FeatureTable.java
@@ -20,10 +20,7 @@ public class FeatureTable implements Iterable<Feature> {
 
   private final Vector<?, ?>[] propertyColumns;
 
-  public FeatureTable(
-      String name,
-      GeometryVector geometryVector,
-      Vector<?, ?>[] properties) {
+  public FeatureTable(String name, GeometryVector geometryVector, Vector<?, ?>[] properties) {
     this(name, null, geometryVector, properties);
   }
 

